docs/userguides/development.md

```diff
@@ -0,0 +1,108 @@
+# Developing a Silverback Application
+
+Silverback applications are excellent for use cases that involve continuously monitoring and responding to on-chain events, such as newly confirmed blocks or contract event logs.
+
+Some examples of these types of applications:
+
+- Monitoring new pool creations, and depositing liquidity
+- Measuring trading activity of popular pools
+- Listening for large swaps to update a telegram group
+
+## Creating an Application
+
+Creating a Silverback Application is easy, to do so initialize the `silverback.SilverbackApp` class:
+
+```py
+from silverback import SilverbackApp
+
+app = SilverbackApp()
+```
+
```

Comment:
```suggestion
The SilverbackApp class handles state and configuration. Through this class, we can hook up event handlers to be executed each time we encounter a new block or each time a specific event is emitted.
```

Reply:
I think a small overview introducing the purpose of this class could be useful here